src/main/java/org/vaadin/pekka/WysiwygE.java

```diff
@@ -61,7 +56,7 @@
 @HtmlImport("bower_components/wysiwyg-e/tools/justify.html")
 @HtmlImport("bower_components/wysiwyg-e/tools/heading.html")
 @HtmlImport("bower_components/wysiwyg-e/tools/blockquote.html")
-public class WysiwygE extends Component implements HasSize, HasStyle, HasValue<WysiwygE, String> {
+public class WysiwygE extends AbstractField<WysiwygE,String> implements HasSize, HasStyle {
```

Comment:
I was planning on extending `AbstractSinglePropertyField` instead. Any reason why not choosing that one ?

Reply:
I had no idea of such a class due to the missing documentation ;-)